pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -36,8 +38,51 @@ def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool,
         self.logger = runtime.logger
         self.working_dir = self.runtime.working_dir
         self.fail_count_name = f'{version}-assembly.{assembly}.count'
+        self.triggered_from_gen_assembly = triggered_from_gen_assembly
+        self.job_run = self.runtime.get_job_run_url()
+
+    async def comment_on_assembly_pr(self):
+        """
+        Comment the link to this jenkins build on the assembly PR if it was triggered automatically
+        """
+        owner = "openshift-eng"
```

Comment:
We can get the owner from doozer_data_path, this would have the nice little benefit of 
if someone runs build-sync from their fork, commenting would still work

Reply:
The owner here represents the base repo against which the PR is being created. Currently the branch name is found correctly because theres a naming convention. If we decided that the name convention remains the same, we should be able to add that feature in. But I think we can keep that as a refinement and create a new ticket for that, since in most cases, the bot will create the PR

Reply:
You're right, I meant something else